cmd/kubeadm/app/apis/kubeadm/apiendpoint.go

```diff
@@ -29,6 +29,9 @@ func APIEndpointFromString(apiEndpoint string) (APIEndpoint, error) {
 	if err != nil {
 		return APIEndpoint{}, errors.Wrapf(err, "invalid advertise address endpoint: %s", apiEndpoint)
 	}
+	if net.ParseIP(apiEndpointHost) == nil {
+		return APIEndpoint{}, errors.Wrapf(errors.New("invalid IP"), "invalid advertise address endpoint IP: %s", apiEndpointHost)
```

Comment:
```suggestion
		return APIEndpoint{}, errors.Errorf("invalid API endpoint IP: %s", apiEndpointHost)
```

please keep the commits squashed to one after this change.

Reply:
Thanks @neolit123 , fixed.